client/verta/verta/_internal_utils/arg_handler.py

```diff
@@ -94,3 +94,22 @@ def maybe(fn, val):
         return fn(val)
     else:
         return None
+
+
+def prepend_slash(path):
```

Comment:
Minor suggestion to name functions like this with the verbs "normalize", "standardize", or "ensure." For example: "standardize_path." This isn't always prepending a slash, only when one which ought to be present is not there.

Reply:
Good point! I've renamed it to `ensure_starts_with_slash()` which is a bit lengthy, but is unambiguous